Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -677,6 +677,34 @@ public final class ManifestLoader: ManifestLoaderProtocol {
             cmd += ["-target", "\(triple.tripleString(forPlatformVersion: version))"]
             #endif
 
+#if os(Windows)
+            // Infer the default flags from the SDKROOT environment variable
+            //
+            // Windows uses a variable named SDKROOT to determine the root of
+            // the SDK.  This is not the same value as the SDKROOT parameter
+            // in Xcode, however, the value represents a similar concept.
+            if let SDKROOT = ProcessEnv.vars["SDKROOT"], let root = try? AbsolutePath(validating: SDKROOT) {
```

Comment:
We have `ManifestResourceProvider` protocol in the manifest loader to plumb such things.

Reply:
Oh, I'll take a look at that.